File: samples/client/petstore/flash/src/main/flex/src/main/flex/io/swagger/client/model/tag.py
package src/main/flex/io.swagger/client/model {


    [XmlRootNode(name="Tag")]
    public class Tag {
    

    

        
        [XmlElement(name="id")]
        
        public var id: Number = None;

    

    

        
        [XmlElement(name="name")]
        
        public var name: String = None;

    

        public function toString(): String {
            var str: String = "Tag: ";
            
            str += " (id: " + id + ")";
            
            str += " (name: " + name + ")";
            
            return str;
        }


}
        

}

File: samples/client/petstore/scala-gatling/src/gatling/scala/io/swagger/client/model/Tag.scala

package io.swagger.client.model


case class Tag (
    _id: Option[Long],
    _name: Option[String]
)
object Tag {
    def toStringBody(var_id: Object, var_name: Object) =
        s"""
        | {
        | "id":$var_id,"name":$var_name
        | }
        """.stripMargin
}
